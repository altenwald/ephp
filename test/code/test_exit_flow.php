<?php

error_reporting(E_ALL);

function early_return() {
    print "we are going to out...\n";
    $data = [];
    if (!isset($data[0])) {
        return(false);
    }
    echo "NOOOOOOO!!!\n";
}

function dying() {
    print "we are going to die!...\n";
    $data = [];
    if (!isset($data[0])) {
        die("dying!");
    }
    echo "NOOOOOOO!!!\n";
}

early_return();
echo "OK\n";
dying();
echo "NO!\n";
