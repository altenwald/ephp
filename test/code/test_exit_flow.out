we are going to out...
OK
we are going to die!...
dying!